project.clj

```diff
@@ -52,6 +52,7 @@
                  ;; Data types, encoding, etc.
                  [byte-streams "0.2.0"]
                  [clj-time "0.11.0"]
+                 [commons-codec "1.9"]
```

Comment:
Did you try using the native Clojure library? (https://github.com/clojure/data.codec)

That's what I've used in the past ...